src/agent/application.hpp

```diff
@@ -119,6 +127,73 @@ class Application : private NonCopyable
      */
     otbrError Run(void);
 
+    /**
+     * Returns the OpenThread controller object the application is using.
+     *
+     * @retval OpenThread controller object.
```

Comment:
My understanding is `@retval` should be used with constants like enums or booleans. For this case `@returns` may fit better.

Reply:
Done.